app/views/items/_item_type_form.html.erb

```diff
@@ -1,13 +1,44 @@
-<% if @item.book? %>
-<h1><%=I18n.t("items.type.book")%></h1>
- <%= render "book_form", item: @item %>
-<% elsif @item.movie? %>
-<h1><%=I18n.t("items.type.movie")%></h1>
- <%= render "movie_form", item: @item %>
-<% elsif @item.game? %>
-<h1><%=I18n.t("items.type.game")%></h1>
- <%= render "game_form", item: @item %>
-<% else @item.other? %>
-<h1><%=I18n.t("items.type.other")%></h1>
- <%= render "other_form", item: @item %>
+<%= form_with(model: @item) do |form| %>
+  <% if @item.errors.any? %>
+    <div style="color: red">
+      <h2><%= pluralize(@item.errors.count, "error") %> prohibited this item from being saved:</h2>
+      <ul>
+        <% @item.errors.each do |error| %>
+          <li><%= error.full_message %></li>
+        <% end %>
+      </ul>
+    </div>
+  <% end %>
+
+  
+  <div class="mb-2">
+    <%= form.label :name, I18n.t("items.form.all.name"), class: "form-label" %>
+    <%= form.text_field :name, class: "form-control" %>
+  </div>
+
+  <div class="mb-2">
+    <%= form.label :description, I18n.t("items.form.all.description"), class: "form-label" %>
+    <%= form.text_field :description, class: "form-control" %>
+  </div>
+
+  <div class="mb-2">
+    <%= form.label I18n.t("items.form.all.max_borrowing_days"), class: "form-label" %>
+    <%= form.number_field :max_borrowing_days, min: 0, max: 365, class: "form-control" %>
+  </div>
+
+  <div class="mb-2">
+    <%= form.label I18n.t("items.form.all.max_reservation_days"), class: "form-label" %>
+    <%= form.number_field :max_reservation_days, min: 0, max: 365, class: "form-control"%>
+  </div>
+  
+  <% if @item.book? %>
+    <%= render "book_form", item: @item, form: form %>
+  <% elsif @item.movie? %>
+    <%= render "movie_form", item: @item, form: form %>
+  <% elsif @item.game? %>
+    <%= render "game_form", item: @item, form: form %>
+  <% else @item.other? %>
+    <%= render "other_form", item: @item, form: form %>
+  <% end %>
+
```

Comment:
Kann man den Submit Button aus den einzelnen Forms nicht auch in diese Datei ziehen? Oder muss der zwangsweise in jeder Type Form liegen?